### DISCUSSION TITLE
Multiple media slots in the same media-controller

### DISCUSSION THREAD

**Discussion by yorudotlife** [muxinc/media-chrome#274] [Q&A] [answered]
Hi,

Is it possible to have multiple videos (aka slot="media") in the same media-controller ?

Best regards,

**Comment by luwes** [comment, marked as answer]
hi @yorudotlife, that's not supported in Media Chrome. what's the use case for this? 

for syncing multiple media elements we will soon have something, but it will group the media elements together in a wrapper element which then can be used as the single media slot. https://github.com/luwes/media-group

see related thread: https://github.com/muxinc/elements/discussions/259

**Reply by yorudotlife** [reply]
Hi Luwes,

Thanks for the super quick response. Thanks.
I am searching a clean way to alternate videos without any form of "sync problem" like a fluid continuous watching experience.

If you had any idea... event with the poor explanation I had given to you... :) I listen to every suggestions.

Best regards,

**Reply by luwes** [reply]
yes, it will require some sync mechanism like the deprecated `MediaController`. I'm working on something like this in my spare time, I'll let you know if I have something ready.

**Reply by luwes** [reply]
I have a early version ready at https://github.com/luwes/media-group, it still needs some improvements and tests but basic functionality is there. let me know if this works, cheers

**Reply by Nodalailama** [reply]
Thanks @luwes, currently, I am gonna follow my approach but I will test your technic.
